org.brlcad.BRL_CAD.yaml

```
@@ -0,0 +1,69 @@
+app-id: org.brlcad.BRL_CAD
+runtime: org.kde.Sdk
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: mged
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+cleanup:
+  - /share/doc
+  - '*.la'
+  - '*.a'
+
+modules:
+  - name: BRL-CAD
+    buildsystem: cmake-ninja
+    builddir: true
+    build-options:
+      cflags: -Wno-error
+      cxxflags: -Wno-error
```

```suggestion
      cflags: -Wno-error=stringop-overflow -Wno-error=format-truncation -Wno-error=inline
        -Wno-error=format-overflow -Wno-error=dangling-pointer -Wno-error=array-bounds -Wno-error=nonnull -Wno-error=maybe-uninitialized
      cxxflags: -Wno-error=stringop-overflow -Wno-error=format-truncation -Wno-error=inline
        -Wno-error=format-overflow -Wno-error=dangling-pointer -Wno-error=array-bounds -Wno-error=nonnull -Wno-error=maybe-uninitialized
```

my bad this actually doesn't work.

upstream should fix it though.